documentation/src/main/asciidoc/topics/con_performance_memory_management.adoc

```diff
@@ -1,23 +1,25 @@
 [id='managing-jvm-memory_{context}']
-= Managing JVM memory
+= JVM memory management with eviction
 
 RAM is a costly resource and usually limited in availability.
 {brandname} lets you manage JVM memory usage to give priority to frequently used data by removing entries from memory.
 
 Eviction:: Controls the amount of data that {brandname} keeps in memory and takes effect for each node.
 
-You can configure whether {brandname} evicts data based on the total number of entries in a cache or by setting a maximum amount of JVM memory that caches can take.
+You have two options to configure whether {brandname} evicts data:
+* By configuring the total number of entries that caches can contain.
+* By setting a maximum amount of JVM memory that caches can take.
 
-With eviction {brandname} also lets you choose between evicting with the Least Frequently Used (LFU) algorithm or the Least Recently Used (LRU) algorithm.
+With eviction, {brandname} lets you choose between evicting with the Least Frequently Used (LFU) algorithm or the Least Recently Used (LRU) algorithm.
 Which algorithm you use depends on whether you store data in the JVM heap or off-heap, has implications beyond the scope of controlling cache staleness.
-However, whether you find LFU a better choice than LRU or the other way around depends entirely on your application semantics and use case.
+The eviction you choose depends entirely on your application semantics and use case.
 
 [IMPORTANT]
 ====
 You should always use eviction with passivation to avoid data inconsistency.
 Because {brandname} evicts entries on a per-node basis, not all nodes evict the same entries.
 ====
 
-In terms of performance, {brandname} does require additional processing to calculate when the size of the cache reaches the configured threshold for eviction.
+In terms of performance, {brandname} requires additional processing to calculate when the size of the cache reaches the configured threshold for eviction.
 To evict an entry from the cache, {brandname} uses the same thread as the write operation that added an entry that exceeds the threshold.
-This additional processing cost is negligible in comparison with the benefit of efficiently managing JVM memory to avoid out of memory exceptions and data loss.
+This additional processing cost outweighs the benefit of efficiently managing JVM memory to avoid out of memory exceptions and data loss.
```

Comment:
@dvagnero Hey, FYI this changes meaning. I'll fix but the change actually says the opposite to what we want to convey.

Reply:
No probs, just so you know.

Reply:
I see! Sorry about that

Reply:
No worries at all. Thanks for the other great edits @dvagnero